app/views/layouts/application.html.erb

```diff
@@ -10,6 +10,10 @@
   </head>
 
   <body>
-    <%= yield %>
-  </body>
-</html>
+  <%= render "layouts/header" %> 
+  <div class="container">
+    <%= yield %> 
+	</div>
```

Comment:
```suggestion
   </div>
```
need fix format your html
example https://www.granneman.com/webdev/coding/formatting-and-indenting-your-html